### PR TITLE
Object#10377 (Data Monitoring) 

### PR DESCRIPTION
**Pull request by BTE164**
Generic interface to set up data monitoring like sensor/log data and mechanism to receive alerts.

### CODE REVIEW

File: 10377.xml [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+			<Item ID="10">
+				<Name>Comparison B </Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the logical statement to compare Data resource against Reference B resource. The statements are encoded as following:
+				0 = No Comparison
```
Comment (akeranen) [inline 10377.xml]: I would recommend to format all these fields as `<value>: <description` to follow the style of IPSO objects. That makes it possible to parse these programmatically.

```suggestion
0: No Comparison
```

**Review by akeranen — COMMENTED** [review]
See also formatting comments in the other PR

File: 10377.xml (hunk 1 above)
Comment (BTE164) [reply 10377.xml]: changed to ':'

**Comment by akeranen** [comment]
Looks good to me, thanks!

**Pull request merged**